website/content/docs/platform/k8s/vso/installation.mdx

```diff
@@ -105,3 +105,56 @@ customresourcedefinition.apiextensions.k8s.io/vaultstaticsecrets.secrets.hashico
+
+To install using Kustomize, download and untar/unzip the latest release from the [Releases Page](https://github.com/hashicorp/vault-secrets-operator/releases).
+```shell-session
+# Example
+$ wget -q https://github.com/hashicorp/vault-secrets-operator/archive/refs/tags/v0.4.2.tar.gz
+$ tar -zxf v0.4.2.tar.gz
+$ cd vault-secrets-operator-0.4.2/
+```
+
+Next install using `kustomize build`:
+```shell-session
+$ kustomize build config/default | kubectl apply -f -
+namespace/vault-secrets-operator-system created
+customresourcedefinition.apiextensions.k8s.io/hcpauths.secrets.hashicorp.com created
+customresourcedefinition.apiextensions.k8s.io/hcpvaultsecretsapps.secrets.hashicorp.com created
+customresourcedefinition.apiextensions.k8s.io/vaultauths.secrets.hashicorp.com created
+customresourcedefinition.apiextensions.k8s.io/vaultconnections.secrets.hashicorp.com created
+customresourcedefinition.apiextensions.k8s.io/vaultdynamicsecrets.secrets.hashicorp.com created
+customresourcedefinition.apiextensions.k8s.io/vaultpkisecrets.secrets.hashicorp.com created
+customresourcedefinition.apiextensions.k8s.io/vaultstaticsecrets.secrets.hashicorp.com created
+serviceaccount/vault-secrets-operator-controller-manager created
+role.rbac.authorization.k8s.io/vault-secrets-operator-leader-election-role created
+clusterrole.rbac.authorization.k8s.io/vault-secrets-operator-manager-role created
+clusterrole.rbac.authorization.k8s.io/vault-secrets-operator-metrics-reader created
+clusterrole.rbac.authorization.k8s.io/vault-secrets-operator-proxy-role created
+rolebinding.rbac.authorization.k8s.io/vault-secrets-operator-leader-election-rolebinding created
+clusterrolebinding.rbac.authorization.k8s.io/vault-secrets-operator-manager-rolebinding created
+clusterrolebinding.rbac.authorization.k8s.io/vault-secrets-operator-proxy-rolebinding created
+configmap/vault-secrets-operator-manager-config created
+service/vault-secrets-operator-controller-manager-metrics-service created
+deployment.apps/vault-secrets-operator-controller-manager created
+$ k get pods -n vault-secrets-operator-system
+NAMESPACE                       NAME                                                         READY   STATUS    RESTARTS   AGE
+vault-secrets-operator-system   vault-secrets-operator-controller-manager-56754d5496-cq69s   2/2     Running   0          1m17s
+```
+
+<Note title="Kustomize does not support all features of the Helm chart">
+  Kustomize does not support all features of the Helm chart.
+  Notably it will not deploy default VaultAuthMethod, VaultConnection or transit related resources.
+  Kustomize also does not support pre-delete hooks that the helm chart uses to cleanup resources
+  and remove finalizers. Please see [`config/samples`](https://github.com/hashicorp/vault-secrets-operator/config/samples)
+  or `config/samples` in the downloaded release artifacts for additional resources.
+</Note>
```

Comment:
Looks like the samples link isn't rendering properly: https://vault-abpoebzmx-hashicorp.vercel.app/vault/docs/platform/k8s/vso/installation#installation-using-kustomize

Played with it locally a bit and I think the text just needs a line separation after/before the <Note> </Note> tags for some reason (see the suggestion below).

We also may not need the first line, since it's a duplicate of the title?

```suggestion
<Note title="Kustomize does not support all features of the Helm chart">

  Notably it will not deploy default VaultAuthMethod, VaultConnection or transit related resources.
  Kustomize also does not support pre-delete hooks that the helm chart uses to cleanup resources
  and remove finalizers. Please see [`config/samples`](https://github.com/hashicorp/vault-secrets-operator/config/samples)
  or `config/samples` in the downloaded release artifacts for additional resources.

</Note>
```